doc/doxygen/install/install-win.doxygen

```diff
@@ -226,7 +226,7 @@
 
         The CMAKE_PREFIX_PATH should hold the path to your Qt5 build directory (see example below). Note that it is NOT the main Qt5 directory, but the subfolder which is named after the toolchain it was build with (e.g. "5.6/msvc2015_64").
         @note If you build Qt5 from source, the build directories will be located at <tt>&lt;path_to_qt&gt;/mkspecs</tt>. Here you will need to choose the matching directory for your architecture and Visual Studio version.
-        @note Example CMAKE_PREFIX_PATH: <tt>C:\dev\qt-5.9\mkspecs\winrt-x64-msvc2017</tt>
+        @note Example CMAKE_PREFIX_PATH: <tt>C:/dev/qt-5.9/mkspecs/winrt-x64-msvc2017</tt>
```

Comment:
Does that work? Worthwhile to double-check.

Reply:
I used `\\` now, to generate `\`, since forward slashes might be confusing for Windows users (even though they work everywhere)